src/main/java/command/AddEventCommand.java

```diff
@@ -0,0 +1,63 @@
+package command;
+
+import ui.Ui;
+
+import storage.TaskList;
+import storage.FileHandler;
+
+import taskmanager.Event;
+
+/**
+ * Command to add an Events task.
+ */
+public class AddEventCommand extends Command{
```

Comment:
missing space (for all subclasses of `Command`, not just this)
```suggestion
public class AddEventCommand extends Command {
```